lib/build_local_glossary.rb

```diff
@@ -0,0 +1,12 @@
+require_relative "./google_sheets_service.rb"
```

Comment:
I'd recommend writing a test for this. It'll document the situation you've discovered and give us more confidence that this doesn't regress.